src/libs/OptionsListUtils.ts

```diff
@@ -208,7 +208,7 @@ type Options = {
     policyReportFieldOptions?: CategorySection[] | null;
 };
 
-type PreviewConfig = {showChatPreviewLine?: boolean; forcePolicyNamePreview?: boolean; showPersonalDetails?: boolean};
+type PreviewConfig = {showChatPreviewLine?: boolean; forcePolicyNamePreview?: boolean; showPersonalDetails?: boolean; lastMessageTextFromReport?: string};
```

Comment:
@suneox Why are we adding the lastMessageTextFromReport again?
lastMessageTextFromReport usage was removed in this PR https://github.com/Expensify/App/pull/41608, and I have tested it in the main branch, our original issue is solved now. Would you also test in the main branch if you can reproduce the original issue?

Reply:
@suneox ⬆️